src/components/PlayerSubmissionForm.js

```diff
@@ -3,22 +3,81 @@ import PropTypes from 'prop-types';
 
 import './PlayerSubmissionForm.css';
 
-const PlayerSubmissionForm = () => {
+const PlayerSubmissionForm = (props) => {
```

Comment:
Just a note, you're getting a warning on the tests just because the tests send different props than you are using in the full app.